apps/coursition/src/app/actions.ts

```diff
@@ -45,8 +49,91 @@ export const createUser = async ({ email, password }: { email: string; password:
     })
 }
 
-export const getInTouch = async (formdata: FormData) => {
-  const data = Object.fromEntries(formdata)
-  console.log({ data })
-  // todo: call resend helper
+export const getInTouch = async (prevState: { message: string }, formdata: FormData) => {
+  const data = Object.fromEntries(formdata) as Record<string, string>
+
+  const { email = '', firstName, lastName, comment } = data
+  return await send({
```

Comment:
**nitpick:** It's better to use async/await only and don't combine it with then/catch.